conf.py

```diff
@@ -83,7 +83,7 @@
 
 # -- Options for linkcheck builder ----------------------------------------
 
-linkcheck_ignore = [r'^https://lsst-lspdev\.ncsa\.illinois\.edu',
+linkcheck_ignore = [r'^https://lsst-lsp-stable\.ncsa\.illinois\.edu',
```

Comment:
I'm not exactly sure what this is doing, but there is now a landing page at that location.